packages/@aws-cdk/aws-cloudfront/lib/web-distribution.ts

```diff
@@ -1112,4 +1146,23 @@ export class CloudFrontWebDistribution extends cdk.Resource implements IDistribu
 
     return originProperty;
   }
+
+  /**
+   * Takes origin shield region from props and converts to CfnDistribution.OriginShieldProperty
+  */
+  private toOriginShieldProperty(originConfig:SourceConfigurationRender): CfnDistribution.OriginShieldProperty | undefined {
+    const originShieldRegion = originConfig.originShieldRegion ??
+    originConfig.customOriginSource?.originShieldRegion ??
+    originConfig.s3OriginSource?.originShieldRegion;
+    if (originShieldRegion) {
+      if (!(/^\w\w\-[a-z]*\-\d$/).test(originShieldRegion)) {
+        throw new Error(`originShieldRegion ${originShieldRegion} is not a valid AWS region.`);
+      }
```

Comment:
If we're removing the region validation from the Distribution, might as well remove it from here for consistency.

Reply:
Ok, missed that one, I removed it and went ahead and made that return use a ternary operator too.